kuma/wiki/models.py

```diff
@@ -1466,7 +1474,7 @@ def nearest_zone(self):
     @cached_property
     def is_zone_root(self):
         zone = self.nearest_zone
-        return bool(zone) and (zone.document in (self, self.parent))
+        return bool(zone) and (zone.document_id in (self.id, self.parent_id))
```

Comment:
👍 if this removes a query

Reply:
@safwanrahman Given your change to the `DocumentNearestZoneJob.fetch` above, which populates the zone's related document, it seems this change is no longer helpful to save a query?

Reply:
I think, `self.parent` would make another call if not compare by id!

Reply:
Ah yes, thanks @safwanrahman!